changelogs/changelog.yaml

```
@@ -409,4 +409,30 @@ releases:
     - 1.5.0.yml
     - 497_mysql_info_returns_connector_name_and_version.yml
     - backport-stable-2-pr-452.yml
-    release_date: '2023-02-08'
\ No newline at end of file
+    release_date: '2023-02-08'
+  1.5.1:
+    changes:
+      major_changes:
+      - The ``community.mysql`` 1.x.y release stream is now effectively End of Life.
+        No more releases will be made. This follows the decision to support each major
+        release for two years. See L(this discussion,https://github.com/ansible-collections/community.mysql/discussions/537)
+        for more info
+      minor_changes:
+      - Integration tests - Add more versions of MariaDB
+      - Integration tests - Add tools to test locally the same as on GHA by using
+        same containers and virtualenv. Custom test containers are published in ghcr.io
+        by this repo's workflows. MySQL/MariaDB are official Docker Hub images.
+      - Integration tests - Carefully verify every component of the tests in the new
+        target 'setup_controller' to ensure expected versions are correct Python,
+        Ansible, connector and MySQL/MariaDB.
+      - Integration tests - New name for many tasks to makes it easier to find failing
+        tests. Rename duplicates. Add name for tasks which doesn't had one, refactor
+        some tests files to better group tests by subject, ...
```

```suggestion
      minor_changes:
      - Integration tests - Add more versions of MariaDB
      - Integration tests - Add tools to test locally the same as on GHA by using
        same containers and virtualenv. Custom test containers are published in ghcr.io
        by this repo's workflows. MySQL/MariaDB are official Docker Hub images.
      - Integration tests - Carefully verify every component of the tests in the new
        target 'setup_controller' to ensure expected versions are correct Python,
        Ansible, connector and MySQL/MariaDB.
      - Integration tests - New name for many tasks to makes it easier to find failing
        tests. Rename duplicates. Add name for tasks which doesn't had one, refactor
        some tests files to better group tests by subject, ...
```
i did miss it anyhow. If it doesn't touch module interfaces for users i think we should remove these fragments?

---

yes, you can but it's not needed in this repo, as trivial will be just ignored

---

you can just remove the fragments before releasing. I'm ok with keeping it as is though. Ideally we shouldn't add any fragments for ci, docs, etc prs that don't touch code